src-docs/src/views/list_group/list_group_extra.tsx

```diff
@@ -25,7 +37,8 @@ export default () => (
     <EuiListGroupItem
       onClick={() => {}}
       wrapText
-      label="Fourth very, very long item with wrapping enabled that will not force truncation"
+      label="Fourth very long item with wrapping enabled, custom props, and will not force truncation."
+      toolTipProps={myToolTipProps}
```

Comment:
See above comment, `children` should not have to be present in the tooltip props. I also don't see a point to extracting this out to a const - let's just inline it
```suggestion
      toolTipProps={{
        delay: 'regular',
        position: 'top',
        title: 'Title of record',
      }}
```